### ISSUE TITLE
FlexibleColumnLayout: Nested interactive controls are not announced by screen readers

### ISSUE THREAD

**Issue by gurkirpalgill** [SAP/ui5-webcomponents#3421]
**Description**
axe DevTools shows these two errors for the FlexibleColumnLayout component

**Isolated Example**
```
<FlexibleColumnLayout
          accessibilityTexts={
            { 
            startArrowRightText: i18n.getText("ONBOARDING_EXPAND_FIRST_COLUMN"), 
            startArrowLeftText: i18n.getText("ONBOARDING_COLLAPSE_FIRST_COLUMN") 
            }
         }
...
```

**Axe tool suggested fix:**
Element has focusable descendants

Also no matter the value of startArrowRightText and startArrowleftText , it alway shows "Expand the first column" and "Collapse the first column"

**Screenshots**

<img width="1917" alt="Screen Shot 2021-06-09 at 2 08 12 PM" src="https://user-images.githubusercontent.com/65705637/121430774-5e6f5d00-c92d-11eb-872e-1f67c5209a72.png">

**UI5 Web Components for React Information**
`@ui5/webcomponents` version:~0.31.9
`@ui5/webcomponents-react` version: ~0.131.7
Operating System: MacOS
Browser: Chrome

**Comment by elenastoyanovaa** [comment]
Hello @gurkirpalgill ,

Before proceeding with the issue processing I would like to give you information, regarding the accessibility support of the UI5 WC.

UI5 Web Components provide the prerequisites for screen reader support based on the HTML, ARIA, and WCAG standards. All screen readers that follow those standards should work fine. Nevertheless, there are deviations in the interpretation depending on the combination of browser and screen reader. UI5 Web Components focus on compliance with the standards, rely on automated checks for accessibility and manual tests with reference testing environments. 

For Screen Reader Support, we recommend using JAWS 2021 + Chrome (latest), and for HTML/ARIA validation the recommended testing tool is Access Assistant (Chrome Extension). UI5 Web Components support other environments to the extend of providing a valid HTML and ARIA implementation following the WCAG standards.

Please note that reference testing environments may change over time to reflect changes in the usage of different browsers, their maintenance period, and increased accessibility compliance.

When reporting issues with different testing environments, please ensure that the issue is not false positive, a real accessibility concern is present and there is an impact over the users. Therefore, we recommend to retest the issue using the mentioned reference testing tools and additionally with plain HTML.

In order to process the issues correctly, we would like to have the following information provided:

Issue description
Reference to the suspected violated accessibility requirement (e.g. Web Content Accessibility Guidelines, WCAG 2.0, BITV 2.0, EN 301 549):
Isolated example:
Steps to reproduce:
UI5 Web Components version:
OS/Platform: {...}:
Browser: {...}:
Testing Tool:

After this information I would like to as you to test with a Screen Reader if nothing is read out really, as assumed by the Axe tool. Please provide info for the exact JAWS version and Browser version.

Thanks in advance!

Elena

**Comment by gurkirpalgill** [comment]
tested with our standard tool NVDA. text is announced but it is always the same text:
"Expand the first column" and "Collapse the first column"
seems to be ignoring the **accessibilityTexts** values
is this the correct usage?
```
<FlexibleColumnLayout
          accessibilityTexts={
            { 
            startArrowRightText: i18n.getText("ONBOARDING_EXPAND_FIRST_COLUMN"), 
            startArrowLeftText: i18n.getText("ONBOARDING_COLLAPSE_FIRST_COLUMN") 
            }
         }
...
```

**Comment by NHristov-sap** [comment]
Hi @gurkirpalgill ,

accessibilityTexts accepts an object, that must have keys you've added, but they must be directly added to the object, not object-in-object as you did it. Please try:

<FlexibleColumnLayout
          accessibilityTexts={ 
            startArrowRightText: i18n.getText("ONBOARDING_EXPAND_FIRST_COLUMN"), 
            startArrowLeftText: i18n.getText("ONBOARDING_COLLAPSE_FIRST_COLUMN") 
         }
...

If you still have issues after this correction, please prepare an isolated example that demonstrates the issue for testing/debugging puprposes.

Best Regards,
Nikolay Hristov

**Comment by fifoosid** [comment]
Hi @gurkirpalgill 

Did you manage to fix this issue on your side?
If so, feel free to close this issue or write to us in order to close it.

**Comment by fifoosid** [comment]
Closing this issue due to inactivity. If you still experience this issue, feel free to reopen it.
